README.md

```diff
@@ -63,7 +65,8 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
+A variable's scope refers to the parts of the code from which the variable can be "seen" and used, or to the other variables and functions it can "see".
 ```
```

Comment:
Good starting definition of scope. Looking for more particular rules like these...

```
1. Variables defined without the var keyword are always placed in the global scope.
2. Variables that are defined using the var keyword are created in the current local scope.
3. Every function and only functions create a new local scope.
4. The current scope includes all outer (enclosing) scopes.
```

Since you weren't here take a look through the [lesson plan](https://github.com/ga-wdi-lessons/js-scope). Let me know if you have any questions, can walk through them this week.